client/components/hikes/HikeFav.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from 'react';
 import axios from 'axios';
+import { Button, List, ListItem, ListItemText, TextField } from '@mui/material';
```

Comment:
MUI suggests using default imports rather than named imports for smaller bundle size. check it out [here](https://mui.com/material-ui/guides/minimizing-bundle-size/)